setup.py

```diff
@@ -64,6 +64,42 @@ def run(self):
 package_data += find_package_data(
     os.path.join('mypyc', 'lib-rt'), ['*.c', '*.h'])
 
+USE_MYPYC = False
+# To compile with mypyc, a mypyc checkout must be present on the PYTHONPATH
+if len(sys.argv) > 1 and sys.argv[1] == '--use-mypyc':
+    sys.argv.pop(1)
+    USE_MYPYC = True
+if os.getenv('MYPYC_USE_MYPYC', None) == '1':
+    USE_MYPYC = True
+
+if USE_MYPYC:
+  MYPYC_BLACKLIST = ('__init__.py',)
+  # Start with all the .py files
+  everything = find_package_data('mypyc', ['*.py'])
+  # Strip out blacklist files
+  mypyc_targets = [x for x in everything if x not in MYPYC_BLACKLIST]
+  # Strip out any test code
+  mypyc_targets = [x for x in mypyc_targets if not x.startswith('test' + os.sep)]
+  # Strip out any mypy files
+  mypyc_targets = [x for x in mypyc_targets if not x.startswith('external' + os.sep)]
```

Comment:
Does it work if you include mypy?

Reply:
Yes, I have modified the setup.py file so that relevant mypy files are included in the compilation unit now.